modules/javascript/pages/ionic.adoc

```diff
@@ -10,28 +10,43 @@ include::partial$_set_page_context_for_javascript.adoc[]
 :param-related: xref:cordova.adoc[] | xref:react.adoc[]
 include::{root-partials}_show_page_header_block.adoc[]
 
+.Support Changes
+[IMPORTANT]
+
+--
+* The Ionic bridging-layer plugin is now open source and community supported.
```

Comment:
no need for bridging-layer, we can just call it plugin as per Ionic terminology

Reply:
Done!